package.json

```
@@ -69,7 +69,7 @@
 		"@typescript-eslint/eslint-plugin": "^8.21.0",
 		"@typescript-eslint/parser": "^8.21.0",
 		"@vercel/node": "^5.0.2",
-		"astro": "^5.1.8",
+		"astro": "^5.7.14",
```

```suggestion
		"astro": "5.7.14",
```
We'd want to remove the "^" in order to fully pin the version, right? Otherwise it could still get updated to 5.8 in the lockfile

---

I have no clue why I left it as a caret constraint--you're right either we'd want a tilde constraint or pin an exact version. I personally like tilde constraint better but an exact version is also fine!